script/Makefile

```diff
@@ -17,7 +17,7 @@ VERSIONFILE := pkg/util/defaults/defaults.go
 VERSION := 1.5.0-SNAPSHOT
 OPERATOR_VERSION := $(subst -SNAPSHOT,,$(VERSION))
 LAST_RELEASED_VERSION := 1.4.0
-RUNTIME_VERSION := 1.7.0
+RUNTIME_VERSION := 1.8.0-SNAPSHOT
```

Comment:
Does this refer to `camel-k-runtime`? shouldn't be paired with `1.9.0-SNAPSHOT` instead?

Reply:
Yes, that's refer to the runtime. We haven't had a 1.8 release of it yet, so that's the version we have in `main` (ref. PR: https://github.com/apache/camel-k-runtime/pull/720).